CDCgov/NEDSS-Modernization
# CNFT2-1154 Prevent editing Published Page

## Description

Adding or editing content of a Page should be blocked unless the page is in `Draft` state. 

## Tickets

* [CNFT2-1154](https://cdc-nbs.atlassian.net/browse/CNFT2-1154)

## Steps to verify

1. Using Swagger, attempt to add a Tab, Section, Subsection, Question to a Published page
2. Verify an exception is thrown and you receive a `Bad Request` message
3. Using Swagger attempt to add a Tab, Section, Subsection, Question to a Draft page
4. Verify no exception is returned and the item is added to the page


[CNFT2-1154]: https://cdc-nbs.atlassian.net/browse/CNFT2-1154?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
